src/main/scala/com/github/sbt/git/JGit.scala

```diff
@@ -29,13 +29,13 @@ final class JGit(val repo: Repository) extends GitReadonlyInterface {
   def branch: String = repo.getBranch
 
   private def branchesRef: Seq[Ref] = {
-    import collection.JavaConverters._
-    porcelain.branchList.call.asScala
+    import scala.jdk.CollectionConverters._
```

Comment:
This is repeated 6 times. I suggest to move the import to the top of the file.

Reply:
Sure, I'll get right on that.

I was thinking the same thing as I was making this conversion. I didn't do this because I haven't worked on the internals here, and wasn't sure if this was a runtime optimization or not.

Reply:
Done.